api/views/course.py

```diff
@@ -126,3 +127,16 @@ def user_stats(self, request, pk=None, category_pk=None):
                 break
 
         return Response({"success_rate": success_rate})
+
+    @action(detail=False, methods=["get"], url_path="registered-students")
+    def registered_students(self, request):
```

Comment:
this function name is misleading. it's returning course_registrations not students.
rename this to `course_registrations`.